Core/Core/Providers/SocialAuth/FacebookAuthProvider.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  FacebookSingInProvider.swift
```

Comment:
Can you please change these names as well after renaming the files? You might need to make this change in all the relevant files like  `SocialAuthView`, `SocialAuthViewModel` etc.